src/Client/ConfigWindow.java

```diff
@@ -1378,6 +1376,40 @@ public void actionPerformed(ActionEvent e) {
     spinnerFatigueNumModel.setValue(98);
     notificationPanelFatigueNotifsSpinner.setModel(spinnerFatigueNumModel);
 
+    JPanel warnHighlightedOnGroundPanel = new JPanel();
+    notificationPanel.add(warnHighlightedOnGroundPanel);
+    warnHighlightedOnGroundPanel.setLayout(
+        new BoxLayout(warnHighlightedOnGroundPanel, BoxLayout.X_AXIS));
+    warnHighlightedOnGroundPanel.setPreferredSize(new Dimension(0, 37));
+    warnHighlightedOnGroundPanel.setAlignmentX(Component.LEFT_ALIGNMENT);
+
+    notificationPanelHighlightedItemTimerCheckbox =
+        addCheckbox(
+            "Warn if one of your highlighted items has been on the ground for more than",
+            warnHighlightedOnGroundPanel);
+    notificationPanelHighlightedItemTimerCheckbox.setToolTipText(
+        "Highlighted items can be configured in the Overlays tab");
+
+    notificationPanelHighlightedItemTimerSpinner = new JSpinner();
+    notificationPanelHighlightedItemTimerSpinner.setMaximumSize(new Dimension(55, 22));
+    notificationPanelHighlightedItemTimerSpinner.setMinimumSize(new Dimension(55, 22));
+    notificationPanelHighlightedItemTimerSpinner.setAlignmentY((float) 0.75);
+    warnHighlightedOnGroundPanel.add(notificationPanelHighlightedItemTimerSpinner);
+    notificationPanelHighlightedItemTimerSpinner.putClientProperty(
+        "JComponent.sizeVariant", "mini");
+
+    JLabel notificationPanelHighlightedItemEndLabel = new JLabel("seconds");
+    warnHighlightedOnGroundPanel.add(notificationPanelHighlightedItemEndLabel);
+    notificationPanelHighlightedItemEndLabel.setAlignmentY((float) 0.9);
+    notificationPanelHighlightedItemEndLabel.setBorder(new EmptyBorder(0, 2, 0, 0));
+
+    // Sanitize JSpinner values
+    SpinnerNumberModel highlightedItemSecondsModel = new SpinnerNumberModel();
+    highlightedItemSecondsModel.setMinimum(0);
+    highlightedItemSecondsModel.setMaximum(630); // 10.5 minutes max
```

Comment:
not sure if its much work or out of scope but would be interesting if between the max timer for mob killed drop and that time placed some warn text so that user knows would be for items either of themselves or of other players killed in wild

Reply:
Think it's gotta be out of scope for this PR, sorry. We just look for highlighted items that are on the ground, and once they are first added by the ground item handler, we put the user-defined timer right there. There won't be any way to know what the source of the drop is from.

Being able to determine ownership of the ground item would be a difficult problem to solve. You could assume that if the ground item first appears under your feet, at that same coordinate, then you "probably" have ownership of it, but it wouldn't be a hardfast rule.

If users want their grounditem timer to encompass even highlighted item drops that they don't have ownership over, they can set the timer lower, to a value they are comfortable with.